src/components/car/CarDetail.js

```diff
@@ -0,0 +1,101 @@
+import { useDispatch, useSelector } from "react-redux";
+import { Link, useSearchParams } from "react-router-dom";
+import { useEffect } from "react";
+import { getCarById } from "../../redux/cars/carsSlice";
+import colorsImg from "../../assets/colors.PNG";
+import arrow from "../../assets/arrow.png";
+
+const CarDetail = () => {
```

Comment:
- [ ] When I try to access a car's details, I get this error:

![image](https://github.com/hmunish/final-capstone-frontend/assets/57421823/887d2746-93c4-4171-807f-f84511f13efe)

The browser's console:


![image](https://github.com/hmunish/final-capstone-frontend/assets/57421823/609a9123-c512-4af0-969a-6f13df1a1a86)


The browser's local storage:

![image](https://github.com/hmunish/final-capstone-frontend/assets/57421823/a9af95d2-5a36-4849-893e-3f51bbad9387)


It seems I'm not authorized to look at the car details even though I'm logged in.

If this is a bug, kindly fix it so your app works as intended.  If this is NOT a bug, it means your app requires special procedures to be run correctly (a `.env` file?, any extra steps required?) if this is the case, kindly add those instructions to your readme so all your users are able to test your app correctly. 👍 

Kindly fix this bug so your app works as intended. 👍